### PR TITLE
Update css selectors, remove elements from the DOM with JS

### PR DESCRIPTION
**Pull request by Rafarel**
Hi there, I just updated the adblocker by using js to remove Ads elements fro the DOM on every page of the website.
So now you'll not have ads on the home page either.
They use to prevent your CSS tricks by using inline style to enforce display block :)

### CODE REVIEW

File: inject.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    '#tall-sidebar-ad',
+    '.short-sidebar-ad-component',
+    '.short-sidebar-ad-top',
+    '.short-sidebar-ad-bottom',
+    '.index-content-ad-wrapper',
+    '.upgrade-push-down',
+    '.r2d2-wrapper'
+]
+
+selectors.forEach(s => {
```
Comment (sshymko) [inline inject.js]: Other than for the logging, the implementation can be simplified to process all selectors altogether.

For example:
```js
document.querySelectorAll(selectors.join(', ')).forEach((el) => el.remove());
```

Comment (Rafarel) [reply inject.js]: Yes of course, I made a quick and dirty code that just works. I can update with your one liner if you prefer :)

Comment (Rafarel) [reply inject.js]: @sshymko I made the update

**Review by sshymko — APPROVED** [review]

**Comment by sshymko** [comment]
@Rafarel
Thanks for incorporating the proposed changes and for submitting the improvement in the first place! 🤝

**Comment by sshymko** [comment]
Tagged [v1.0.3](https://github.com/upscalesoftware/chesscom-adblock/releases/tag/1.0.3)

**Pull request merged**